net/packet/packet.go

```diff
@@ -180,7 +183,7 @@ func (q *Parsed) decode4(b []byte) {
 			}
 			q.Src = q.Src.WithPort(binary.BigEndian.Uint16(sub[0:2]))
 			q.Dst = q.Dst.WithPort(binary.BigEndian.Uint16(sub[2:4]))
-			q.TCPFlags = TCPFlag(sub[13]) & 0x3F
+			q.TCPFlags = TCPFlag(sub[13]) & 0xFF
```

Comment:
Isn't this the same as removing `& 0xFF` entirely? You're masking off none bits of an 8-bit value.

Reply:
whoops, yeah. I was reviewing it in wireshark which groups the prior byte into a uint16 and just brain-o'ed it here.

will remove.